<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1022</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="header">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="prev_Button">
          <property name="toolTip">
           <string>返回</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="logoLabel">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>&lt;b&gt;Music&lt;b&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next_Button">
          <property name="toolTip">
           <string>前进</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchLine">
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="userPix">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>登录</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>未登录 ▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minButton">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="toolTip">
           <string>最小化</string>
          </property>
          <property name="text">
           <string>_</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="toolTip">
           <string>关闭</string>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="main">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="mainHorizontalLayout" stretch="0,3,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>198</width>
             <height>530</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="recommandLabel">
              <property name="text">
               <string>推荐</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="recommandList">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>110</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>110</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="myLabel">
              <property name="text">
               <string>我的音乐</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="myList">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>110</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>110</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="collectLabel">
              <property name="text">
               <string>收藏与创建的歌单</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="collectList">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>110</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>110</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true">QTabWidget::pane { border: 2px solid rgb(76, 76, 113);}QTabBar::tab {\n                                 border: 0px solid #C4C4C3;border-bottom-color: #C2C7CB; border-top-left-radius: 0px; border-top-right-radius: 0px;\n                                 width: 00px;height: 00px;padding: 0px;} QTabBar::tab:selected{color:rgb(255,255,255);background-color:rgb(76, 76, 113);\n                                    border-color: rgb(76, 76, 113);}</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="detailView">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>627</width>
             <height>530</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="playWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="previousButton">
          <property name="toolTip">
           <string>上一首</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="toolTip">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseButton">
          <property name="toolTip">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="toolTip">
           <string>下一首</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentTime">
          <property name="toolTip">
           <string>当前时间</string>
          </property>
          <property name="text">
           <string>01:09</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="timeSlider">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>5</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="countTime">
          <property name="toolTip">
           <string>总时间</string>
          </property>
          <property name="text">
           <string>03:04</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="volume">
          <property name="toolTip">
           <string>音量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="noVolume">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>静音</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volumeSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="single">
          <property name="toolTip">
           <string>单曲循环</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="repeat">
          <property name="toolTip">
           <string>列表循环</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playlist">
          <property name="toolTip">
           <string>播放列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>prev_Button</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_prev_page_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_Button</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_next_page_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_prev_song_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_next_song_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>696</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_play_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_pause_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlist</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_playList_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1001</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>1010</x>
     <y>683</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_volumeSlider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>878</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>861</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noVolume</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_noVolume_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volume</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>slot_volume_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>702</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_prev_page_clicked()</slot>
  <slot>slot_next_page_clicked()</slot>
  <slot>slot_prev_song_clicked()</slot>
  <slot>slot_next_song_clicked()</slot>
  <slot>slot_play_clicked()</slot>
  <slot>slot_pause_clicked()</slot>
  <slot>slot_playList_clicked()</slot>
  <slot>slot_volumeSlider_valueChanged(int)</slot>
  <slot>slot_noVolume_clicked()</slot>
  <slot>slot_volume_clicked()</slot>
 </slots>
</ui>
